.github/workflows/build-ci.yml

```diff
@@ -107,9 +93,5 @@ jobs:
                     $([[ "${{ matrix.mode }}" == ignore-php-req ]] && echo ' --ignore-platform-req=php+')
             -   name: "Run tests"
                 run: |
-                  if [ "${{ matrix.mongodb }}" = "Atlas" ]; then
-                    export MONGODB_URI="mongodb://127.0.0.1:27017/"
-                  else
-                    export MONGODB_URI="mongodb://127.0.0.1:27017/?replicaSet=rs"
-                  fi
-                  ./vendor/bin/phpunit --coverage-clover coverage.xml
+                  export MONGODB_URI="mongodb://127.0.0.1:27017/?replicaSet=rs"
+                  ./vendor/bin/phpunit --coverage-clover coverage.xml --exclude-group atlas-search
```

Comment:
By filtering on the group, there is less tests, and I don't get crashes in GitHub Actions.